spec/lucky/text_response_spec.cr

```diff
@@ -22,9 +22,22 @@ describe Lucky::TextResponse do
       print_response(context, status: nil)
       context.response.status_code.should eq 300
     end
+
+    it "prints no body with a head call" do
+      context = build_context("HEAD")
+      print_response_with_body(context, "Body", "text/plain", nil)
+      context.request.method.should eq "HEAD"
+      context.request.body.to_s.should eq ""
+      context.response.status_code.should eq 200
+      context.response.headers["Content-Type"].should eq "text/plain"
+    end
   end
 end
 
 private def print_response(context : HTTP::Server::Context, status : Int32?)
-  Lucky::TextResponse.new(context, "", "", status: status).print
+  print_response_with_body(context, "", "", status)
+end
+
+private def print_response_with_body(context : HTTP::Server::Context, body : String = "", content_type : String = "", status : Int32? = nil)
```

Comment:
Since this is only used by the one spec, I think you can make `body` required (no default) and remove the content_type and status